<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Status</class>
 <widget class="QWidget" name="Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QGroupBox" name="daemonsListBox">
     <property name="title">
      <string>Daemons</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Administration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="confMonitorButton">
        <property name="text">
         <string>Config Monitor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confRobotButton">
        <property name="text">
         <string>Config robot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="deviceListBox">
     <property name="title">
      <string>Devices</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mqtt broker</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="mqttStatusLabel">
        <property name="text">
         <string>[ERROR]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="dbStatusLabel">
        <property name="text">
         <string>[ERROR]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QGroupBox" name="sensorsListBox">
     <property name="title">
      <string>Sensors list</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
